CHANGELOG.md

```diff
@@ -5,7 +5,27 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 ### Changed
-- Avoid the print of highest logger levels when the exception is re-thrown. [PR 1187](https://github.com/pgjdbc/pgjdbc/pull/1187)
+
+### Added
+
+### Fixed
+
+## [42.2.3] (2018-07-04)
+### Changed
+- Reduce severity level of log messages in Driver, PgConnection, ConnectionFactoryImpl [PR 1187](https://github.com/pgjdbc/pgjdbc/pull/1187)
+- Deprecate Fastpath API [PR 903](https://github.com/pgjdbc/pgjdbc/pull/903)
+- Support parenthesis in {oj ...} JDBC escape syntax [PR 1204](https://github.com/pgjdbc/pgjdbc/pull/1204)
+- ubenchmark module moved pgjdbc/benchmarks repository due to licensing issues [PR 1215](https://github.com/pgjdbc/pgjdbc/pull/1215)
+- Improved CONTRIBUTING.md [PR 951](https://github.com/pgjdbc/pgjdbc/pull/951)
```

Comment:
The release notes (CHANGELOG.md) should include only **Notable changes**, I think that something like "Improved CONTRIBUTING.md" is not "Notable"

Reply:
I think [How to submit a bug report](https://github.com/pgjdbc/pgjdbc/blob/master/CONTRIBUTING.md#how-to-submit-a-bug-report) is notable for end-users.

Reply:
Ok, fair enough :slightly_smiling_face: 

In that case something like "Include section how to submit a bug report in CONTRIBUTING.md" is more descriptive.